content/200-concepts/100-components/02-prisma-client/053-client-extensions/200-extension-examples.mdx

```diff
@@ -1,17 +1,41 @@
 ---
 title: 'Examples'
-metaTitle: 'Prisma Client Extensions | Examples'
-metaDescription: 'Extend the functionality of Prisma Client: Examples'
+metaTitle: 'Prisma Client extensions | Prisma- & Community-built extensions and examples'
+metaDescription: 'Prisma Client extensions | Prisma- & Community-built extensions and examples'
```

Comment:
```suggestion
metaTitle: 'Prisma Client extensions | Prisma & Community built extensions and examples'
metaDescription: 'Prisma Client extensions | Prisma & Community built extensions and examples'
```
This looks better to my eyes but no clue if it's right. take it or leave it

Reply:
As a German, any two words that make one word that are separated by a space is a "Deppenleerzeichen". But that's German... so no idea what you are allowed to do in English :D

Reply:
You can do this in English, but the syntax of having two the prepended "Prisma-" before the ampersand struck my eye as odd

Reply:
I would suggest to rephrase this entirely. Also: is it intentional that `metaTitle` and `metaDescription` are identical here? I would imagine this  looks odd when shared on social also misses an opportunity to share more context about this page via the `metaDescription`. I would suggest using different phrases for each and using the `metaDescription` to provide more context. 

```suggestion
metaTitle: 'Prisma Client extensions | Shared packages & examples'
metaDescription: 'Explore the Client extensions that have been built by Prisma and its community'
```